must-gather/collection-scripts/gather_collection_resources

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+source ./common
+
+# Expect base collection path as an argument
+BASE_COLLECTION_PATH=$1
+
+# Use PWD as base path if no argument is passed
+if [ "${BASE_COLLECTION_PATH}" = "" ]; then
+    BASE_COLLECTION_PATH=$(pwd)
+fi
+
+NAMESPACE="openshift-logging"
+
+CLO_COLLECTION_PATH="$BASE_COLLECTION_PATH/cluster-logging"
+collector_folder="$CLO_COLLECTION_PATH/collector"
+
+check_collector_connectivity() {
+  local pod=$1
+  echo "--Connectivity between $pod and elasticsearch" >> $collector_folder/$pod
+
+  es_host=$(oc -n $NAMESPACE  get pod $pod  -o jsonpath='{.spec.containers[0].env[?(@.name=="ES_HOST")].value}')
```

Comment:
I don't believe this is relevent any longer in light of LF

Reply:
@ewolinetz WDYT? I transfered all these from logging-dump.

Reply:
if we gather the LF CR then we should have this information (though it is technically still TP). logging-dump is still a little out of date with respect to our current deployment processes, i had updated it simply to gather additional information that we required for debugging -- no addressing of stale functionality

Reply:
I will keep this as is for now and we can wipe out stale things in a separate PR.